lib/update.php

```diff
@@ -340,6 +361,28 @@ public function vimeography_addon_update_message($plugin_basename, $plugin_data)
   }
 
   /**
+   * Add a reminder to update an expiring key
+   * for the installed Vimeography theme / addon.
+   *
+   */
+  public function vimeography_key_expiring_message() {
+    $today = date("Y-m-d");
+
+
+    if($expiration_date < $today){
+      echo '<div class="notice notice-warning"><p>';
+        printf( __('Heads up! Your %1$s license is expiring soon! For continued support, activate your plugin <a href="http://www.vimeography.com/%2$s">here</a>', 'vimeography'), $this->expiring_addon->product_name, $this->expiring_addon->plugin_name );
```

Comment:
For the message, how about this instead:
>To continue to receive updates and support for `plugin_name`, click here to renew your license.

I'll add the dismiss button, no problem.